### PR TITLE
CADC-12805 - support additional Range blueprint test cases, remove deprecated imp import 

### PR DESCRIPTION
**Pull request by SharonGoliath**

### CODE REVIEW

File: caom2utils/caom2utils/caom2blueprint.py [hunk 1]
```diff
@@ -2587,70 +2578,60 @@ def _get_metrics(self, current):
...
+                    if aug_function is not None:
+                        # if the WCS is described with a Function, cutouts can be supported, so specify an axis
+                        aug_naxis_index = _to_int(self._get_from_list(f'Chunk.{label}Axis', index))
+                    self.logger.debug(f'Creating function {label} CoordAxis1D for {self.uri} from blueprint')
+            else:
+                aug_naxis = CoordAxis1D(axis=aug_axis, error=aug_error, range=aug_range)
+                self.logger.debug(f'Creating range {label} CoordAxis1D for {self.uri} from blueprint')
+
+        self.logger.debug(f'End {label} axis construction from blueprint.')
+        return aug_naxis, aug_naxis_index
```
Comment (andamian) [inline caom2utils/caom2utils/caom2blueprint.py:2634]: WCS is something that I want to understand so I can make sense of this code.

File: caom2utils/caom2utils/tests/test_collections.py [hunk 2]
```diff
@@ -172,20 +174,31 @@ def _vos_client_meta(subject, uri):
...
-            except UnicodeDecodeError:
-                hdulist = fits.open(fqn, memmap=True, lazy_load_hdus=True)
-                hdulist.verify('fix')
-                hdulist.close()
-                headers = [h.header for h in hdulist]
+            if '.fits' in fqn:
+                # during operation, want to use astropy on FITS files
+                # but during testing want to use headers and built-in Python file
+                # operations
+                from urllib.parse import urlparse
```
Comment (andamian) [inline caom2utils/caom2utils/tests/test_collections.py:181]: This should be in the import section

**Review by andamian — APPROVED** [review]

File: caom2utils/caom2utils/tests/test_collections.py (hunk 2 above)
Comment (SharonGoliath) [reply caom2utils/caom2utils/tests/test_collections.py:181]: Moved.

File: caom2utils/caom2utils/caom2blueprint.py (hunk 1 above)
Comment (SharonGoliath) [reply caom2utils/caom2utils/caom2blueprint.py:2634]: You and me both.

**Pull request merged**